SteamP2PTransport/SteamP2PTransport.cs

```diff
@@ -121,17 +121,17 @@ public override ulong ServerClientId
 
         public override void DisconnectLocalClient()
         {
-            if (LogHelper.CurrentLogLevel <= LogLevel.Developer) LogHelper.LogInfo("SteamP2PTransport - DisconnectLocalClient");
```

Comment:
@TwoTenPvP NetworkLog.CurrentLogLevel is internal so I had to use NetworkingManager.Singleton.LogLevel instead. Is this intended?